pkg/output/jsonl.go

```diff
@@ -50,16 +50,12 @@ type JSONData struct {
 type MatcherResults struct {
 	FileType   *scanner.FileType   `json:"filetype,omitempty"`
 	Parameters []scanner.Parameter `json:"parameters,omitempty"`
-	Errors     []MatcherResult     `json:"errors,omitempty"`
-	Infos      []MatcherResult     `json:"infos,omitempty"`
-	Secrets    []MatcherResult     `json:"secrets,omitempty"`
-}
-
-type MatcherResult struct {
```

Comment:
I think we should keep this struct in output, so that we can evolve it later. Maybe there will be more keys than 'Name' and 'Match' later on, and we could add them here.